src/main/xerus/monstercat/api/APIConnection.kt

```diff
@@ -74,23 +74,50 @@ class APIConnection(vararg path: String) : HTTPQuery<APIConnection>() {
 	fun getTracks() =
 		parseJSON(TrackResponse::class.java)?.results
 	
+	fun postLogin(username: String, password: String) =
+		post(HttpPost(uri).apply {
+			setHeader("Accept", "application/json")
+			setHeader("Content-type", "application/json")
+			entity = StringEntity("{\"email\":\"$username\",\"password\":\"$password\"}")
+		})
+	
+	fun postLogout() =
+		post(HttpPost(uri).apply {
+			setHeader("Accept", "application/json")
```

Comment:
please set these headers for every request and remove this method

Reply:
This should be removed indeed; to logout all we need to do is remove the connect.sid

Reply:
Removed in a81ff488b772515fd42a35feb308eb07e3cb503e